spark_client/cli/service_account_registry.py

```diff
@@ -170,7 +170,7 @@ def create_service_account_registry_parser(parser: ArgumentParser):
         if maybe_service_account is None:
             raise NoAccountFound(input_service_account.id)
 
-        maybe_service_account.configurations.log(logging.info)
+        maybe_service_account.configurations.log(logging.critical)
```

Comment:
as before

Reply:
explanation same as shared in the previous comment.